### PR TITLE
Added Dimonds and Ownership to cli and refactored subparser[Issue #20]

### LINKED ISSUE #20: Hook up the Diamond, DiamondLoupeFacet, OwnershipFacet CLIs to "game7ctl"
Currently, if you want to use those CLIs, you have to call them as:

```
python -m game7ctl.DiamondLoupeFacet verify-contract -h
```

These should be accessible directly from the `game7ctl` invocation as:

```
game7ctl diamond-loupe verify-contract -h
```

### PR DESCRIPTION
**Pull request by muddi900**
Fixes #20

### CODE REVIEW

File: game7ctl/game7ctl/cli.py [hunk 1]
```diff
@@ -2,9 +2,17 @@
 
 from .core import generate_cli as core_generate_cli
 from .InventoryFacet import generate_cli as inventory_generate_cli
+from .DiamondLoupeFacet import generate_cli as dloupe_generate_cli
+from .DiamondCutFacet import generate_cli as dcut_generate_cli
+from .OwnershipFacet import generate_cli as own_generate_cli
 from .version import VERSION
 
 
+def add_subparser(cmd_name: str, subparser, cli_gen: callable):
```
Comment (zomglings) [inline game7ctl/game7ctl/cli.py:11]: @muddi900 : Seems like an unnecesary abstraction - you aren't really saving anything with this `add_subparser` function.

But if you do want to keep it, I don't have a super strong opinion.

I would suggest:
1. `subparser` should have type annotation: `argparse.ArgumentParser`
2. Fix type annotation on `cli_gen` using `typing.Callable` - https://mypy.readthedocs.io/en/stable/kinds_of_types.html#callable-types-and-lambdas

**Comment by zomglings** [comment]
@muddi900 : Looks ok to me, could you please bump the version to 0.0.2?

**Comment by zomglings** [comment]
@muddi900 : Could you merge `main`, as well? And run `black` on your code?

**Comment by muddi900** [comment]
The branch is rebased, the code is reformatted and the version number is bumped.

**Review by ogarciarevett — CHANGES_REQUESTED** [review]
Please aim to the  "develop" branch

**Review by zomglings — APPROVED** [review]
lgtm

@ogarciarevett : I changed target branch to `develop` myself. Merging now.

**Pull request merged**